### PR TITLE
chore(dev): fix asset upload script

### PR DESCRIPTION
**Pull request by jlzhao27**
example end to end run: https://github.com/wandb/weave/actions/runs/5955305109/job/16153659956

image

### CODE REVIEW

**Review by bitcoinfinancier — APPROVED** [review]

File: weave/frontend/bundle.sh [hunk 1]
```diff
@@ -6,9 +6,9 @@ SHA=$(cat $SCRIPT_DIR/sha1.txt)
 cd $SCRIPT_DIR/..
 tar --exclude='frontend/bundle.sh' --exclude='frontend/build.sh' -cvzf /tmp/$SHA.tar.gz frontend
 
-if [[ "${IS_RELEASE_ENV}" == "true" && "${CI}" == "true" ]]; then
+if [[ "${CI}" == "true" && -n "${GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
     echo "Uploading bundle to cloud storage..."
-#    gsutil cp /tmp/$SHA.tar.gz gs://wandb-cdn-prod/weave/*
+    gsutil cp /tmp/$SHA.tar.gz gs://wandb-cdn-prod/weave/*
```
Comment (vanpelt) [inline weave/frontend/bundle.sh:11]: Get rid of the star here or replace it with `$SHA.tar.gz`

**Review by vanpelt — APPROVED** [review]

File: .github/workflows/upload-assets.yaml [hunk 2]
```diff
@@ -34,12 +34,16 @@ jobs:
...
-
+          service_account: ${{ secrets.WORKLOAD_IDENTITY_SERVICE_ACCOUNT }}
+      - id: 'upload-and-push'
+        if: ${{ steps.build.outputs.UPLOAD_ASSETS == 'true' }}
+        run: |
+          ./weave/frontend/bundle.sh
+          git config --global user.name 'Weave Build Bot'
+          git config --global user.email weave@wandb.com
+          git commit -a -m 'chore(bot): update frontend bundle sha'
+          git push
```
Comment (vanpelt) [inline .github/workflows/upload-assets.yaml:49]: I think there's a built in action step that will update instead of this, but if this works 👍

Comment (jlzhao27) [reply .github/workflows/upload-assets.yaml:49]: hmm, I did a quick google search and didn't see any official ones. I saw some third party ones but I was skeptical. If you got one in mind, let me know

File: weave/frontend/bundle.sh (hunk 1 above)
Comment (jlzhao27) [reply weave/frontend/bundle.sh]: fixed it to use actual path

**Pull request merged**